ui/css/treeherder-details-panel.css

```diff
@@ -422,3 +422,32 @@ div.performance-panel {
   /* This is the same value as m-1 */
   margin: 0.25rem;
 }
+
+.performance-panel-data td, th {
```

Comment:
Would you mind trying to get this layout using the `bootstrap` `utilities` like `pl-1`,`pr-1` for padding and `text-nowrap` classes instead of custom CSS?  I realize some custom CSS is necessary, but we are trying to minimize it.

We also have lots of helper utilities classes in `treehrder-custom-styles.css` like `small-text` is `font-size: 12`, etc.